16/umbraco-cms/reference/content-delivery-api/additional-preview-environments-support.md

```diff
@@ -6,110 +6,20 @@ description: >-
 # Additional preview environments support
 
 {% hint style="warning" %}
-The contents of this article have not yet been verified or updated for Umbraco 15.
+The original contents of this article, explaining how to change the backoffice UI, does not apply anymore to Umbraco 15 and above and has been removed.
+The way to achieve this beyond Umbraco 14 is to create and register an extension to modify the backoffice UI preview button.
```

Comment:
Can we link to anywhere to show how this is done?  Even if we don't have a specific article to show how to modify the backoffice UI preview button, at least to a page generally talking about client-side extensions.

Reply:
Thanks @AndyButland  for the review I have added the link 😊